modules/nf-core/purecn/run/meta.yml

```diff
@@ -0,0 +1,185 @@
+name: "purecn_run"
+description: Run PureCN workflow to normalize, segment and determine purity and ploidy
+keywords:
+  - copy number alteration calling
+  - hybrid capture sequencing
+  - targeted sequencing
+  - DNA sequencing
+tools:
+  - "purecn":
+      description: "Copy number calling and SNV classification using targeted short read sequencing"
+      homepage: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      documentation: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      tool_dev_url: "https://github.com/lima1/PureCN"
+      doi: "10.1186/s13029-016-0060-z"
+      licence: "Artistic-2.0"
+      args_id: "$args"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - intervals:
+      type: file
+      description: |
+        BED file of target intervals, generated from IntervalFile.R
+      pattern: "{*.bed,*.txt}"
+  - coverage:
+      type: file
+      description: Coverage file generated from Coverage.R
+      pattern: "*.txt"
+  - vcf:
+      type: file
+      description: |
+        Normal panel in VCF format, used to calculate mapping bias
+      pattern: "*.vcf.gz"
+  - normaldb:
+      type: file
+      description: |
+        Normal panel in RDS format, generated from NormalDB.R
+      pattern: "*.rds"
+  - genome:
+      type: string
+      description: Genome build
+
+output:
+  - csv:
+      type: file
+      description: |
+        CSV file containing copy number calls
+      pattern: "*.csv"
+  - variants_csv:
+      type: file
+      description: |
+        CSV file containing SNV calls
+      pattern: "*_variants.csv"
+  - pdf:
+      type: file
+      description: |
+        PDF file containing copy number plots
+      pattern: "*.pdf"
+  - rds:
+      type: file
+      description: |
+        RDS file containing copy number calls
+      pattern: "*.rds"
+  - amplification_pvalues_csv:
+      type: file
+      description: |
+        CSV file containing amplification p-values
+      pattern: "*_amplification_pvalues.csv"
+  - chr_pdf:
+      type: file
+      description: |
+        PDF file containing chromosome plots
+      pattern: "*_chromosomes.pdf"
+  - seg:
+      type: file
+      description: |
+        Segmentation file generated from DNAcopy.R
```

Comment:
I'm not sure this is actually true for all segmentation methods. Can you look at the PureCN code and see whether `_dnacopy.seg` is appended regardless of the segmentation method used?

Reply:
It is, if I'm not wrong: https://github.com/lima1/PureCN/blob/d90f3f0e80fde4b1ba70ad6e99d1870baf708157/inst/extdata/PureCN.R#L458